src/main/java/ru/hogwarts/school/controller/StudentController.java

```diff
@@ -17,53 +15,43 @@
 public class StudentController {
     private final StudentService studentService;
 
-    @Autowired
     public StudentController(StudentService studentService) {
         this.studentService = studentService;
     }
 
-
-    @Operation(summary = "Создать нового студента")
-    @PostMapping("/create")
-    public ResponseEntity<Student> createStudent(@RequestBody Student student) {
+    @PostMapping("/")
+    public ResponseEntity<Student> createStudent(@RequestBody Student student) throws DatabaseAccessException {
         Student createdStudent = studentService.create(student);
         return ResponseEntity.ok().body(createdStudent);
     }
-    @Operation(summary = "Получить студента по ID")
-    @GetMapping("/read/{id}")
+    @GetMapping("/{id}")
     public ResponseEntity<Student> getStudent(@PathVariable long id) {
         Student student = studentService.read(id);
         if (student == null) {
             return ResponseEntity.notFound().build();
         }
         return ResponseEntity.ok().body(student);
     }
-    @Operation(summary = "Обновить существующего студента ")
-    @PutMapping("/update")
+    @PutMapping("/")
     public ResponseEntity<Student> updateStudent(@RequestBody Student student) {
         Student updatedStudent = studentService.update(student);
         if (updatedStudent == null) {
             return ResponseEntity.notFound().build();
         }
         return ResponseEntity.ok().body(updatedStudent);
     }
-    @Operation(summary = "Удалить студента по ID")
-    @DeleteMapping("/delete/{id}")
+    @DeleteMapping("/{id}")
     public ResponseEntity<Void> deleteStudent(@PathVariable long id) {
         boolean deleted = studentService.delete(id);
         if (!deleted) {
             return ResponseEntity.notFound().build();
         }
         return ResponseEntity.noContent().build();
     }
-    @Operation(summary = "Фильтровать студентов по возрасту")
-    @GetMapping("/filterByAge/{age}")
-    public ResponseEntity<List<Student>> filterStudentsByAge(@PathVariable int age) {
+
+    @GetMapping("/filter")
```

Comment:
```suggestion
    @GetMapping(params = "age")
```
не должно быть никаких слов в пути